src/inmanta/server/services/dryrunservice.py

```diff
@@ -157,6 +157,16 @@ async def dryrun_list(self, env: data.Environment, version: Optional[int] = None
             {"dryruns": [{"id": x.id, "version": x.model, "date": x.date, "total": x.total, "todo": x.todo} for x in dryruns]},
         )
 
+    @handle(methods_v2.list_dryruns, env="tid")
+    async def list_dryruns(self, env: data.Environment, version: int) -> List[DryRun]:
+        model = await data.ConfigurationModel.get_version(environment=env.id, version=version)
+        if model is None:
+            raise NotFound("The requested version does not exist.")
+
+        dryruns = await data.DryRun.get_list(order_by_column="date", order="DESC", environment=env.id, model=version)
```

Comment:
This is very inefficient (getting all the content out of the database to then just forget about it), but I don't know if we care.

@bart wdyt?